pkg/util/liveconfig/hive.go

```diff
@@ -116,3 +116,10 @@ func (p *prod) AdoptByHive(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func (p *prod) OCMValidClientIDs() ([]string, error) {
+	// TODO: Replace with RP Live Service Config (KeyVault)
+	// until then, IDs can be added to RP-Config `configuration.validOCMClientIDs: '["id1", "id2"]'
```

Comment:
I'd like some input here. We ultimately want to be able to support multiple valid OCM clientIDs. The way this is currently implemented we would have one ENV variable set to a bash array `["id1", "id2"]` but I'm not sure if this will work having to convert the RP-Config from a yaml to json and then into the Configuration struct. 

Do we have a better way to accomplish this goal today?